docs/SOP.md

```diff
@@ -23,6 +25,45 @@ This document contains standard operating procedures (SOPs) for the OBO Foundry
+1. Ontology scope. The new ontology must present some use cases where it will be relevant to life-sciences.  Was the ontology developed using expert input or trusted scientific sources representative of the consensus in its target domain of knowledge? If the ontology was developed for a very specific purpose or community, representation and consensus need not be broad; however, this scope should be clearly stated.
+2. Terms with the new ontology prefix. The new terms provided by the ontology should not already be available in an available OBO Foundry ontology. If there is an overlap of terms, justification should be provided. There must not be a term with the same label available. There should not be another OBO Foundry ontology whose scope covers those terms and if there is there should be some evidence of contacting that OBO Foundry ontology.
+3. Correct use of imported terms. Does the ontology accurately reuse terms from other OBO ontologies?
+Are imported terms in appropriate hierarchies? That is, has the import of the term preserved its upper-level alignment?
+Are any additional axioms used for these terms correct in both a technical (e.g. passes reasoning) and substantive sense?
+4. Basic review of axiomatic patterns:
+Are existential restrictions used correctly? Typical mistakes include “R some (A and B and C)” to mean “(R some A and R some B and R some C)”
+Are axioms generally highly complex? If so, we should review a handful to ensure they are as intended.
+5. Appropriate use of [object properties](https://www.w3.org/TR/2004/REC-owl-semantics-20040210/#owl_ObjectProperty). Examples of incorrect usage include those based on some interpretation of the label of the object property but not actually fitting the property definition or domain and range. A typical example of incorrect usage is R some (A and B and C) to mean R some A and R some B and R some C.
+6. Responsiveness to fixing changes. A willingness to fix any identified issues during the review must be demonstrated. Issues expected to be addressed should be added in TODO mode in the GitHub issue. The time limit for addressing these is 2 months.
+
+<a name="OAE"></a> 
+### Ontology Acceptance Email 
+
+Once a new ontology has been accepted, the following should be used as boiler plate for letting them know and what are the next steps they should take.
+
+"Thank you again for your ontology submission to the OBO Foundry. We are happy to inform you that your ontology (YOURID) has been accepted following discussion in the OBO Operations Committee meeting, DD-MM-YYYY. Before we can add it to the OBO ontology registry you need to complete the following steps.
+
+Create a metadata record for your ontology to be included in the registry
+1. Create a new file in this directory,  https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology, called YOURID.md (there is an “Add file” button in the top right):
```

Comment:
is there an image following the :? If not, remove the :

Reply:
agree with change